mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(priceBeforeTax) {
+  let priceWithTax = priceBeforeTax * 1.2;
```

Comment:
This confused me to begin with as to why you were multiplying the price by 1.2. One of the reasons is that I have no idea what the significance of that number is. In the industry we'd refer to this as a "magic number". It's considered better to treat it as a named constant defined somewhere in your code. Some people/organisations will capitalize shared constants, something like:

```
const TAX_RATE = 0.2;

function calculateSalesTax(price) {
    let tax = price * TAX_RATE;
    return price + tax;
}
```

Reply:
Multiplied by 1.2 to get the price plus the tax.